apitest/docs/api-beta-test-guide.md

```diff
@@ -0,0 +1,492 @@
+        --bitcoinPath=<bitcoin-core-home>/bin
+```
+
+If your bitcoin-core binaries are not statically linked to your BerkleyDB library, you can specify the path to it
+with the `–-berkeleyDbLibPath=<path>` option:
+```
+$ ./bisq-apitest --apiPassword=xyz
+        --supportingApps=bitcoind,seednode,arbdaemon,alicedaemon,bobdaemon
+        --shutdownAfterTests=false
+        --bitcoinPath=<bitcoin-core-home>/bin
+        --berkeleyDbLibPath=<lib-berkleydb-path>
+```
+
+Alternatively, you can specify any or all of these bisq-apitest options in a properties file located in
+`apitest/src/main/resources/apitest.properties`.
+
+In this example, a beta tester uses the `apitest.properties` below, instead of ./bisq-cli options.
+```
+supportingApps=bitcoind,seednode,arbdaemon,alicedaemon,bobdaemon
+apiPassword=xyz
+shutdownAfterTests=false
+bitcoinPath=/home/beta-tester/path-to-my-bitcoin-core/bin
+```
+
+Start up the test harness with without command options:
+```
+$ ./bisq-apitest
+```
+
+If you edit  `apitest.properties`, do not forget to re-build the source.  You do not need to do a full clean and
+build, or run tests.  The following build command should finish quickly.
+```
+$ ./gradlew build :apitest:installDaoSetup -x test
+```
+
+You should see the test harness startup bitcoin-core and other Bisq daemons in your console, run a
+`bitcoin-cli getwalletinfo` command, and generate a regtest btc block.
+
+After the test harness tells you how to shut it down by entering `^C`, the test harness is ready to use.
+
+## Running Trade Simulation Script
+
+_Warning:  again, it is assumed the beta tester has a reasonably fast machine, or the scripted wait times -- for
+the other side to perform his step in the protocol, and for btc block generation and asynchronous processing
+of new btc blocks by test daemons -- may not be long enough._
+
+### System Requirements
+
+Same as described at the top of this document, but your bitcoin-core’s `bitcoin-cli` binary must be in the system
+`PATH`.  (The script generates regtest blocks with it.)
+
+### Description
+
+The regtest trade simulation script `apitest/scripts/trade-simulation.sh` is a useful introduction to the Bisq Api.
+The bash script’s output is intended to serve as a tutorial, showing how the CLI can be used to create payment
+accounts for Bob and Alice, create an offer, take the offer, and complete a trade protocol.
+(The bash script itself is not intended to be as useful as the output.)  The output is generated too quickly to
+follow in real time, so let the script complete before studying the output from start to finish.
+
+The script takes four options:
+```
+-d=<direction>          The trade direciton, BUY or SELL.
+-c=<country>            The two letter country code, US, FR, AT, RU, etc.
+-f=<fixed-price>        The offer’s fixed price.
+    OR  (-f and -m options mutually exclusive, use one or the other)
+-m=<margin-from-price>  The offer’s margin (%) from market price.
```

Comment:
```suggestion
-m=<margin-from-price>  The offer’s margin (%) below the current market price.
```

Reply:
Just realized this probably changes whether the offer is to buy or to sell. So disregard.